pulp_docker/app/tasks/sync_stages.py

```diff
@@ -326,30 +346,39 @@ class InterrelateContent(Stage):
         Relate each item in the input queue to objects specified on the DeclarativeContent.
         """
         async for dc in self.items():
+
             if dc.extra_data.get('relation'):
-                if type(dc.content) is ManifestList:
-                    self.relate_manifest_list(dc)
-                elif type(dc.content) is ManifestBlob:
-                    self.relate_blob(dc)
-                elif type(dc.content) is ImageManifest:
-                    self.relate_manifest(dc)
-
-            configured_dc = dc.extra_data.get('config_relation')
-            if configured_dc:
-                configured_dc.content.config_blob = dc.content
-                configured_dc.content.save()
+                self.relate_manifest_to_list(dc)
+            elif dc.extra_data.get('blob_relation'):
+                self.relate_blob(dc)
+            elif dc.extra_data.get('config_relation'):
```

Comment:
Can a single unit have relations to multiple types of other objects? If so, I think these need to be ifs, not elifs.

Reply:
my thinking was to create a singe relation type per unit, so elif is correct here